explainer.md

```diff
@@ -186,8 +194,6 @@ There are several open questions to address:
 1. Whether this API should be restricted to top-level frames.
```

Comment:
Either here or in a separate change, it would be useful to file GitHub issues for these open questions so they can be discussed/resolved in the group.  (And replace this list with a list of links to those issues.)

Reply:
Will address in a separate PR.

Reply:
Leaving this open so it's not forgotten :)

Reply:
Put up a separate PR for this: https://github.com/webscreens/site-initiated-mirroring/pull/11